lab/services/nginx/nginx.conf

```diff
@@ -0,0 +1,74 @@
+load_module modules/ngx_http_datadog_module.so;
+
+worker_processes 1;
+
+events {
+    worker_connections  1024;
+}
+
+error_log stderr debug;
+
+http {
+    # datadog_sample_rate 1.0;
```

Comment:
The content of `lab/` will change as features are added to the module. At first I didn't include `lab/` in the repo, because it seemed more like local dev work than source code, but then I reasoned that it would be useful to anybody working on the repo. Then there's the question of "noise" resulting from the regular hacking that will occur to its configuration.

What do you think is a good way to manage this?

Reply:
I'm a fan of having placeholders for things that'd normally be toggled in this "lab", when turning on and off certain features or test conditions. Most of them should eventually turn into tests, but sometimes it's just probing a specific issue.
I wouldn't try to predict what's useful or not for the future, so just leave in the parts you have found useful for now.

Reply:
ok